src/Coordination/KeeperConstants.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace DB
+{
+
+const std::string keeper_system_path = "/keeper";
+const std::string keeper_api_version_path = keeper_system_path + "/api_version";
+
+enum class KeeperApiVersion : uint8_t
+{
+    V0 = 0, // ZooKeeper compatible version
+    V1      // added FilteredList request
```

Comment:
What do you think about more descriptive names like `ZOOKEEPER_COMPATIBLE` and `WITH_FILTERED_LIST`? We use such names for ClickHouse native protocol revisions, see `ProtocolDefines.h`

Reply:
I'm okay with that, it's explicit and more clear from the code